cudax/include/cuda/experimental/__stf/graph/graph_ctx.cuh

```diff
@@ -517,6 +517,23 @@ public:
     cudaGraphDebugDotPrint(get_graph(), filename.c_str(), flags);
   }
 
+  template <typename T>
+  auto transfer_host(cuda::experimental::stf::logical_data<T>& ldata)
+  {
+    using valT = typename owning_container_of<T>::type;
+    valT out;
```

Comment:
No need for the `using` - you're using the name just once.

```suggestion
    typename owning_container_of<T>::type out;
```

Reply:
done